tjreports/administrator/tjreports.php

```diff
@@ -34,14 +34,15 @@
 }
 
 $document = JFactory::getDocument();
-
 // Load js assets
 $tjStrapperPath = JPATH_SITE . '/media/techjoomla_strapper/tjstrapper.php';
```

Comment:
@sneha27 Please add this font awesome without strapper and remove this strapper code.

Reply:
Are there any other files loaded using strapper?

Reply:
@manojLondhe No, we are not using this to load any files.
@sneha27 Also check site tjreports.php and remove the same strapper code from there as well.